0b.md

```diff
@@ -0,0 +1,273 @@
+NIP-0B
+======
+
+On-Behalf of - Simple Sub-Key Management
+-----
+
+`draft` `optional`
+
+This NIP defines a simple way for an Identity Sub-Key management system, including active, inactive and revoked public keys, for publishing On-Behalf of a master identity.
+
+### Motivation
+
+The state of art now has two methods for enabling sub-key management:
+- [NIP-26](https://github.com/nostr-protocol/nips/blob/master/26.md): Multiple key delegation and no revocation (just time bound)
+- [NIP-41](https://github.com/nostr-protocol/nips/blob/master/41.md): More complete single key delegation and revocation (complex)
```

Comment:
All links to NIP-41 are broken. This doesn't exist.

Reply:
fixed 👍